src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -29,7 +30,7 @@ public SarifValidatorTests()
         {
             _jsonSchemaFilePath = Path.Combine(Environment.CurrentDirectory, JsonSchemaFile);
             string schemaText = File.ReadAllText(JsonSchemaFile);
-            _schema = SchemaReader.ReadSchema(schemaText);
+            _schema = SchemaReader.ReadSchema(schemaText, JsonSchemaFile);
```

Comment:
The new version of the `SchemaReader` takes then name of the schema file so it can include it in the `Result` objects it produces when there's an error reading the schema.